src/content/docs/apm/agents/nodejs-agent/extend-your-instrumentation/nodejs-v1-custom-instrumentation-legacy.mdx

```diff
@@ -1,145 +0,0 @@
----
```

Comment:
@bizob2828 it looks like there's still a reference to this file in the left navigation.

<img width="706" alt="image" src="https://github.com/user-attachments/assets/af118fdc-6f84-4c3c-a1bb-65d8ba4f170e">

Lines 509 and 510 should be removed from `src/nav/apm.yml`

```
              - title: Node.js v1 custom instrumentation
                path: /docs/apm/agents/nodejs-agent/extend-your-instrumentation/nodejs-v1-custom-instrumentation-legacy
```

Reply:
ah ok. ill do that